### PR TITLE
Objects Automated Test - LPS-141680 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-141680

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase [hunk 1]
```diff
@@ -1408,12 +1408,29 @@ definition {
...
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
 
-		// TODO LPS-141680 CannotDeletePublishedObjectViaApi pending implementation
+		ObjectAdmin.addObjectRelationshipViaAPI(
+			objectName_1 = "CustomObject",
+			objectName_2 = "CustomObject",
+			relationshipLabel = "Relationship",
+			relationshipName = "relationship",
+			relationshipType = "oneToMany");
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase:1423]: Hi @andreefarias
You don't need to create a relationship for this test. You need to create a field for that

**Review by evanilson-santana — CHANGES_REQUESTED** [review]
Hi @andreefarias,

Please, check my comments and make the necessary changes.

Let me know if you have any questions.

Thanks.

**Review by evanilson-santana — APPROVED** [review]

**Comment by evanilson-santana** [comment]
ci:forward

**Pull request closed without merging**